src/templates/app/user/metatrader.jsx

```diff
@@ -180,6 +180,18 @@ const Metatrader = () => (
                                 </a>
                             </div>
                         </div>
+                        <div className='mt-sidebar-button'>
+                            <div className='small-icon'>
+                                <a href={it.url_for('metatrader/download')}>
+                                    <img src={it.url_for('images/pages/metatrader/dashboard/img-app-mac@2x.png')} />
+                                </a>
+                            </div>
+                            <div className='mt-link'>
+                                <a href='https://trade.mql5.com'>
```

Comment:
@arash-binary  please add `target` and `rel` attributes

Reply:
Sure

Reply:
@ashkanx ` target='_blank' rel='noopener noreferrer'` is it right?

Reply:
yes